Source/WebCore/page/FocusController.cpp

```diff
@@ -123,14 +140,20 @@ Node* FocusNavigationScope::firstChildInScope(const Node& node) const
 {
     if (is<Element>(node) && isFocusScopeOwner(downcast<Element>(node)))
         return nullptr;
-    return node.firstChild();
+    auto* first = node.firstChild();
+    while (isOpenPopoverWithInvoker(first))
```

Comment:
Maybe this should be renamed to isInPopoverShowingState?
The fact it may haves an invoker isn't of interest here.

Reply:
I think here we care both about the popover showing as well as having an invoker.